### PR TITLE
Fixed restore_step for new training

### PR DESCRIPTION
**Pull request by m-toman**
last_epoch for _LRScheduler should be -1 if not resuming a previous optimization process
see https://pytorch.org/docs/stable/_modules/torch/optim/lr_scheduler.html

### CODE REVIEW

**Comment by erogol** [comment]
It doen't make a difference on account of this line below in the scheduler we have. But thx pointing out.

```step = max(self.last_epoch, 1)```

**Comment by m-toman** [comment]
Actually this came up for me when starting from scratch because I got this exception:
https://github.com/pytorch/pytorch/blob/master/torch/optim/lr_scheduler.py#L19-L23
So for the constructor it has to be -1.

**Comment by erogol** [comment]
It is fixed by now,
